Feature Management: add documentation (#78108)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

| `externalServiceAccounts` | Automatic service account and token setup for plugins                                        |
| `panelTitleSearchInV1`    | Enable searching for dashboards using panel title in search v1                               |
| `ssoSettingsApi`          | Enables the SSO settings API                                                                 |

> ## Configure Feature Management

Reviewer: Sentence casing

```suggestion
## Configure feature management
```

> Feature Management is an Administration page that allows admin users to visualize and toggle the feature flags available in their Grafana instance, for more information refer to [Feature Management](/docs/grafana/latest/administration/feature-management/").

Reviewer: ```suggestion
**Feature Management** is an Administration page that allows admin users to visualize and toggle the feature flags available in their Grafana instance, for more information, refer to [Feature Management](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/feature-management/).
```

> Feature Management is an Administration page that allows admin users to visualize and toggle the feature flags available in their Grafana instance, for more information refer to [Feature Management](/docs/grafana/latest/administration/feature-management/").

Reviewer 2: ```suggestion
**Feature management** is an **Administration** page that allows admin users to visualize and toggle the feature flags available in their Grafana instance. For more information, refer to [Feature Management](/docs/grafana/<GRAFANA_VERSION>/administration/feature-management/).
```

> By default, feature toggles are in read-only mode. Granting admin users the ability to alter the states of feature toggles requires configuring Grafana with the optional [allow_editing](/docs/grafana/latest/setup-grafana/configure-grafana#allow_editing), [update_webhook](/docs/grafana/latest/setup-grafana/configure-grafana#update_webhook) and [update_webhook_token](/docs/grafana/latest/setup-grafana/configure-grafana#update_webhook_token) settings.

Reviewer: ```suggestion
By default, feature toggles are in read-only mode.
Granting admin users the ability to alter the states of feature toggles requires configuring Grafana with the optional [allow_editing](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#allow_editing), [update_webhook](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#update_webhook) and [update_webhook_token](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#update_webhook_token) settings.
```

> By default, feature toggles are in read-only mode. Granting admin users the ability to alter the states of feature toggles requires configuring Grafana with the optional [allow_editing](/docs/grafana/latest/setup-grafana/configure-grafana#allow_editing), [update_webhook](/docs/grafana/latest/setup-grafana/configure-grafana#update_webhook) and [update_webhook_token](/docs/grafana/latest/setup-grafana/configure-grafana#update_webhook_token) settings.

Reviewer 2: ```suggestion
By default, feature toggles are in read-only mode. Granting admin users the ability to alter the states of feature toggles requires configuring Grafana with the optional [allow_editing](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana#allow_editing), [update_webhook](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana#update_webhook), and [update_webhook_token](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana#update_webhook_token) settings.
```
